helpers/Courses/createCourse.js

```diff
@@ -3,11 +3,11 @@ const { v4 } = require("uuid");
 
 async function createCourse(tutorId, courseName, initSessionPrice, hourlySessionPrice) {
   await Courses.create({
-    courses_id: v4(),
-    course_name: courseName,
-    initial_session_price: initSessionPrice, 
-    session_hourly_rate: hourlySessionPrice,
-    tutor_id: tutorId
+    id: v4(),
```

Comment:
Here you don't pass the course id, but you do in the appointments for creation. Is there a reason why?